.circleci/config.yml

```diff
@@ -400,6 +352,8 @@ jobs:
           name: Make sure all templates can be simulated.
           command: |
             (cd src/improvements && just simulate-all-templates)
+      - notify-failures-on-main:
```

Comment:
NIT: Just to make sure when you meant: 
>  I'm adding a slack notification to the template regression tests.
This is the only notification you want to add here?